tasks/test_1.py

```diff
@@ -0,0 +1,15 @@
+from selenium import webdriver
+import time
+
+driver = webdriver.Chrome(executable_path="chromedriver")
+driver.get("https://www.youtube.com/@zusmani78/videos")
+
+scroll_distance = 2000
+i = 0
+
+while True:
```

Comment:
so the loop will keep running infinitely. it will never stop. i am expecting better solutions from you.

Reply:
ok